src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
> It's pointless to declare something in a namespace and then require the name space to be used to call it.

Generally, I totally agree with this. `using` statement is hackish and should be avoided whenever possible.

> If namespacing requires a using statement in order to use the literal designator, the literals should be moved out of the units namespace

But I also think that some protection is still better than nothing, and local `using` is a lesser evil than total absence of the namespace. Even STL encourages namespaces for literals:
```c++
void f() {
  using namespace std::chrono_literals;
  ...
}
```
If I knew a proper way of prepending, I wouldn't suggest this.
Anyways, literals are out of scope of this PR.